pkg/proto/nbcontract/v1/kubebinaryconfig.proto

```diff
@@ -11,4 +11,10 @@ message KubeBinaryConfig {
    
   //privately cached kube binary url
   string private_kube_binary_url = 3;  
+
+  //full path to the "pause" image. Used for --pod-infra-container-image.
+  string pod_infra_container_image_url = 4;
```

Comment:
used in containerd template. Karpenter hardcodes this, but could break things with RP, so we should add this to contract.